### PR TITLE
Hotfix (06.01.2022)

### PR DESCRIPTION
**Pull request by NeonKirill**
Fixed order for `register_consumer()` as it broke dependencies

### CODE REVIEW

File: neon_mq_connector/connector.py [hunk 1]
```diff
@@ -247,9 +247,10 @@ def create_mq_connection(self, vhost: str = '/', **kwargs):
         return pika.BlockingConnection(parameters=self.get_connection_params(vhost, **kwargs))
 
     def register_consumer(self, name: str, vhost: str, queue: str,
```
Comment (NeonDaniel) [inline neon_mq_connector/connector.py:249]: Add some unit tests that look like calls to this method that exist currently so we can test for similar breakage in the future

**Review by NeonDaniel — CHANGES_REQUESTED** [review]
We should have tests that reproduce any discovered issues to avoid hitting this same problem in the future

**Comment by NeonDaniel** [comment]
Looks like there is a problem in new tests?
```
    def test_stable_register_consumer_args(self):
        try:
            # Required connector.register_consumer() arguments order:
            # name: str, vhost: str, queue: str,
            # callback: callable, on_error: Optional[callable] = None,
            # auto_ack: bool = True
            self.connector.register_consumer("test_consumer",
                                             self.connector.vhost,
                                             'test',
                                             self.connector.callback_func_1,
                                             self.connector.default_error_handler,
                                             False)
            self.assertIsInstance(self.connector.consumers['test_consumer'], ConsumerThread)
            self.assertEqual(self.connector.consumers['test_consumer'].queue, 'test')
            self.assertEqual(self.connector.consumers['test_consumer'].exchange, '')
        except Exception as ex:
>           self.fail(f'Registering consumer failed: ***ex***')
E           AssertionError: Registering consumer failed: 'ConsumerThread' object has no attribute 'exchange'
```

**Comment by NeonKirill** [comment]
> Looks like there is a problem in new tests?
> 
> ```
>     def test_stable_register_consumer_args(self):
>         try:
>             # Required connector.register_consumer() arguments order:
>             # name: str, vhost: str, queue: str,
>             # callback: callable, on_error: Optional[callable] = None,
>             # auto_ack: bool = True
>             self.connector.register_consumer("test_consumer",
>                                              self.connector.vhost,
>                                              'test',
>                                              self.connector.callback_func_1,
>                                              self.connector.default_error_handler,
>                                              False)
>             self.assertIsInstance(self.connector.consumers['test_consumer'], ConsumerThread)
>             self.assertEqual(self.connector.consumers['test_consumer'].queue, 'test')
>             self.assertEqual(self.connector.consumers['test_consumer'].exchange, '')
>         except Exception as ex:
> >           self.fail(f'Registering consumer failed: ***ex***')
> E           AssertionError: Registering consumer failed: 'ConsumerThread' object has no attribute 'exchange'
> ```

Some issue with github environment, as locally causing no issues, omitted this check for now

**Review by NeonDaniel — COMMENTED** [review]
Just needs license header on added test file and then good to go

**Review by NeonDaniel — APPROVED** [review]

**Pull request merged**